### PR TITLE
Add state container, abstract out styling

### PR DESCRIPTION
**Pull request by eldrickb**
Abstracts the state and views of `VoxettaApp.js` into their own components.
- `StateContainer.js` is a component that sits on top of the app. This component centralizes global business & rendering logic by catching events as they bubble up, making calls to JS services, changing its internal state, and passing the necessary properties down to...
- `ViewContainer.js`, which is a component that determines the view to display based on properties passed passed to it by `StateContainer`. 

notes:
- also abstracts out styling into their own folder; this change was made before I abstracted out the state container 

edit: sorry for the massive PR it's mostly the styling files

### CODE REVIEW

File: ui/public/styles/normalize.css [hunk 1]
```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```
Comment (mmallett) [inline ui/public/styles/normalize.css:1]: Can you get this from a CDN?

File: ui/src/components/VoxettaPrompts.js [hunk 2]
```diff
@@ -99,40 +80,49 @@ export class VoxettaPrompts extends LitElement {
...
             case 'FAILURE':
                 return html`<p><b>Prompt failed to load.</b></p>`;
             case 'FINISHED':
                 return html`<p>Your work session is finished.</p>`;
         }
     }
 
+    handleResetPrompts() {
+        promptApi.resetAllPromptsUnread();
+        // .then(() => this.getNewPrompt());
```
Comment (mmallett) [inline ui/src/components/VoxettaPrompts.js:102]: remove

File: ui/public/styles/normalize.css (hunk 1 above)
Comment (eldrickb) [reply ui/public/styles/normalize.css:1]: yes, should I do that instead?

File: ui/public/styles/normalize.css [hunk 3]
```diff
@@ -0,0 +1,351 @@
...
+
+body {
+    margin: 0;
+}
+
+/**
+   * Render the `main` element consistently in IE.
+   */
+
+main {
```
Comment (ashleygranitto) [inline ui/public/styles/normalize.css:31]: Does this file overrride the styling within every component, i.e. if I had a <main> with `display: flex`, would the display render as `flex` or `block`?

File: ui/public/styles/normalize.css [hunk 4]
```diff
@@ -0,0 +1,351 @@
...
+   * 1. Show the overflow in Edge.
+   */
+
+button,
+input {
+    /* 1 */
+    overflow: visible;
+}
+
+/**
```
Comment (ashleygranitto) [inline ui/public/styles/normalize.css:182]: nit: the spacing in this file is a little weird--aligning the * might make it a little cleaner

File: ui/src/components/UserIcon.js [hunk 5]
```diff
@@ -19,38 +19,47 @@ import {LitElement, html, css} from 'lit-element';
...
     render() {
         return html`
-            <mwc-icon-button 
-                icon="account_circle"
-                @click=${this.handleUserIconClick}
-            </button>
+            <div @click=${this.handleUserIconClick}>
+                <mwc-icon-button icon="account_circle"> </mwc-icon-button>
+
+                <span>
```
Comment (ashleygranitto) [inline ui/src/components/UserIcon.js:55]: I believe this is only for the non-mobile version--maybe make just the icon button the default?

Comment (eldrickb) [reply ui/src/components/UserIcon.js:55]: it's `display: hidden` by default

File: ui/public/styles/normalize.css (hunk 4 above)
Comment (eldrickb) [reply ui/public/styles/normalize.css:182]: will run prettier

File: ui/public/styles/normalize.css (hunk 3 above)
Comment (eldrickb) [reply ui/public/styles/normalize.css:31]: it'll be lowest in priority; it wont' override your styles

**Pull request merged**